rapids_config.cmake

```diff
@@ -25,6 +25,8 @@ else()
       "Could not determine RAPIDS version. Contents of VERSION file:\n${_rapids_version_formatted}")
 endif()
 
+set(rapids-cmake-repo jameslamb/rapids-cmake)
+set(rapids-cmake-branch fmt-and-spdlog)
```

Comment:
```suggestion
```

TODO: remove this right before merging

Reply:
removed in https://github.com/rapidsai/rmm/pull/1678/commits/4135b68d5cc1673636ed8a114b2275734146c229